site/src/pages/DeploySettingsPage/NotificationsPage/storybookUtils.ts

```diff
@@ -0,0 +1,216 @@
+import type { Meta } from "@storybook/react";
+import {
+	notificationDispatchMethodsKey,
+	systemNotificationTemplatesKey,
+} from "api/queries/notifications";
+import type { DeploymentValues, SerpentOption } from "api/typesGenerated";
+import {
+	MockNotificationMethodsResponse,
+	MockNotificationTemplates,
+	MockUser,
+} from "testHelpers/entities";
+import {
+	withAuthProvider,
+	withDashboardProvider,
+	withDeploySettings,
+	withGlobalSnackbar,
+} from "testHelpers/storybook";
+import type { NotificationsPage } from "./NotificationsPage";
+
+// Extracted from a real API response
+export const mockNotificationsDeploymentOptions: SerpentOption[] = [
+	{
+		name: "Notifications: Dispatch Timeout",
```

Comment:
This will go out of sync; is there a way we can lint this?

Reply:
The only check we have here is the type, as we do for the other mocks as well, but I know it is not sufficient. I've been thinking about how we could improve our mocks and keep them in sync. One option that came to mind was to use a script that communicates with the API and generates the mocks. However, this would require more investigation and is beyond the scope of this pull request. What do you think about this idea? Do you see a simpler way we could achieve better results?

Reply:
Perfectly fine to do as a follow-up, yup. Your idea sounds good 👍